xcb.c

```diff
@@ -233,8 +235,14 @@ void grab_pointer_and_keyboard(xcb_connection_t *conn, xcb_screen_t *screen, xcb
         }
     }
 
-    if (tries <= 0)
+    /* After trying for 10000 times, i3lock will display an error message
+     * for 2 sec prior to terminate. */
+    if (tries <= 0) {
+        pam_state = STATE_PAM_FAIL;
+        redraw_screen();
+        usleep(2000000);
```

Comment:
Use sleep() instead of usleep() when you’re specifying seconds, it’s easier to read.

Also, does anything speak against showing this for just 1s? I’d like to avoid situations where the screen doesn’t change for so long that people think everything is okay and just walk away, not paying much attention to the error message.